modules/mailer/mailer.go

```diff
@@ -29,24 +30,32 @@ type Message struct {
 }
 
 // NewMessageFrom creates new mail message object with custom From header.
-func NewMessageFrom(to []string, from, subject, htmlBody string) *Message {
-	log.Trace("NewMessageFrom (htmlBody):\n%s", htmlBody)
+func NewMessageFrom(to []string, from, subject, body string) *Message {
+	log.Trace("NewMessageFrom (body):\n%s", body)
 
 	msg := gomail.NewMessage()
 	msg.SetHeader("From", from)
 	msg.SetHeader("To", to...)
 	msg.SetHeader("Subject", subject)
 	msg.SetDateHeader("Date", time.Now())
 
-	body, err := html2text.FromString(htmlBody)
+	var plainBody string
+	var err error
+	if strings.Contains(body[:100], "<html>") {
```

Comment:
At this stage it should be a complete HTML file. Anyway I removed it but still check for HTML when sending as plain to generate warning should there be a <html> in the body.